Blackjack.Tests/ControllerShould.cs

```diff
@@ -45,13 +38,28 @@ public void GivePlayer2Cards_WhenGameStarts()
         [Fact]
         public void GivePlayerAnotherCard_WhenPlayerHits()
         {
-            _mockInput.Setup(x => x.ReadLine()).Returns("1"); 
+            _mockInput.Setup(x => x.ReadLine()).Returns("1");
+            _mockDeck.Setup(x => x.DealCard()).Returns(It.IsAny<Card>());
             var mockPlayer = new Mock<IPlayer>();
             _players.Add(mockPlayer.Object);
 
             _controller.Play();
 
+            _mockDeck.Verify(x => x.DealCard(), Times.Exactly(3));
             mockPlayer.Verify(x => x.ReceiveCard(It.IsAny<Card>()), Times.Exactly(3));
         }
+
+        [Fact]
+        public void GivePlayer2MoreCards_WhenPlayerHitsTwice() 
+        // TODO: failing - need to fix logic so player can make a choice more than once - what is the end condition (while loop)?

```

Comment:
Think about what would end a real-life game of blackjack. You can replicate those conditions for the loop.